org.sofproject.ui/src/org/sofproject/ui/wizards/NewAudioDevNodeWizard.java

```diff
@@ -59,8 +59,8 @@ public void init(IWorkbench workbench, IStructuredSelection selection) {
 
 		super.addPages();
```

Comment:
@edominia please enhance your commit description. A good start is to read:

https://chris.beams.io/posts/git-commit/

So, commit subject line should be smaller (around 50 chars) and contain a tag that will let people know
the area for which this commit is intended.

Then in the commit body explain WHY the change is needed. 

So, your commit should look like this:

E.g 
```
ui: Change wizard name

Use 'configuration' instead of 'connection' to better reflect
the real use.
```

I know this is a trivial commit and I will insist on doing proper
commit messages *now* even for simpler changes because
it is a good practice. 

Please keep in mind that a commit message should always tell WHY
the commit is needed and after explain (if there is any need) how the
things are done.